src/Utilities/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -681,6 +681,9 @@ function saturation_adjustment(e_int::DT, ρ::DT, q_tot::DT) where DT
     T, converged = find_zero(
       T -> internal_energy_sat(T, ρ, q_tot) - e_int,
       T_1, T_2, SecantMethod(); xatol=DT(1e-3), maxiters=10)
+      if !converged
```

Comment:
@trontrytel Codecov may be failing if this segment of the code is not being tested.

Reply:
I think it's still better to have it